src/apify_client/_http_client.py

```diff
@@ -76,6 +77,13 @@ def _parse_params(params: dict | None) -> dict | None:
             # Our API needs lists passed as comma-separated strings
             elif isinstance(value, list):
                 parsed_params[key] = ','.join(value)
+            elif isinstance(value, datetime):
+                utc_aware_dt = value.replace(tzinfo=timezone.utc)
```

Comment:
This only replaces the timezone without actually computing the difference, right? 

So if I e.g. submit a `datetime.now()` to this method, I would only get runs from two hours in the future (I'm on Central European Summer Time).

<img width="803" height="102" alt="image" src="https://github.com/user-attachments/assets/58e17d7c-011f-4d9e-bb36-3ee6ebe2b1ec" />